app/views/messages/_message.html.erb

```diff
@@ -63,7 +63,7 @@ end
         ><%= format_for_copying message.content_text %></div>
         <div
           class="hidden"
-          data-speaker-target="text <%= message.assistant? && "assistantText" %>"
+          data-speaker-target="text <%= message.assistant? ? "assistantText" : "userText" %>"
```

Comment:
I actually don't think this is helping. I thought I could use the presence userText to distinguish between the two tricky cases but I'm almost certain it's not helping. I should eventually revert this when I come up with a better strategy.